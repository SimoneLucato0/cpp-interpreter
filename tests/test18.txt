(21+43) > 99 != ("Success" != "Failure") != (73 >= 75)