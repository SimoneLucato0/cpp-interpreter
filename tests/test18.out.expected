LEFT_PAREN ( null
NUMBER 21 21.0
PLUS + null
NUMBER 43 43.0
RIGHT_PAREN ) null
GREATER > null
NUMBER 99 99.0
BANG_EQUAL != null
LEFT_PAREN ( null
STRING "Success" Success
BANG_EQUAL != null
STRING "Failure" Failure
RIGHT_PAREN ) null
BANG_EQUAL != null
LEFT_PAREN ( null
NUMBER 73 73.0
GREATER_EQUAL >= null
NUMBER 75 75.0
RIGHT_PAREN ) null
EOF  null
